### ISSUE TITLE
Compiled DLLs?

### ISSUE THREAD

**Issue by MichaelGatesDev** [elfprince13/FreeBuild-Torque-OSG-Plugins#1]
I am interested in trying your plugins with the project but I can not get the main project nor this project to build to include in OSG. If you could compile it for me, that would be greatly appreciated. I know the project is very very old at this point.

**Comment by elfprince13** [comment]
Which project specifically do you mean?

At this point I don't even remember what version of OpenSceneGraph I was working with for these plugins.

**Comment by MichaelGatesDev** [comment]
I'm working on a project where we've tried to convert DIF (torque interior format) to a 3d format such as obj. Unfortunately we've only had luck with a few of the interiors.  We used [this awesome project](https://github.com/HiGuyMB/Dif) to do that. 

While searching around the web, I came across this project and decided that to see if OSG would be able to load our interior format. Unfortunately there are no compiled DLL files for me to drag+drop into a OSG to test that, and I am unable to compile the project myself because it yells at me about dependencies and I am far from a C programmer. I was hoping if you were able to compile it and upload it to the releases section, I could test it out.

In the event you're not able (don't have time/interest/etc) I am just going to look myself to see what you did differently [here](https://github.com/elfprince13/FreeBuild-Torque-OSG-Plugins/blob/master/osgPlugins/bsp/DIFReader.cpp) compared to [here](https://github.com/HiGuyMB/Dif/blob/master/src/dif/objects/interior.cpp)
